conversant/personas/travel-advisor/config.json

```diff
@@ -0,0 +1,70 @@
+{
+    "chatbot_config": {
+        "max_context_examples": 10,
+        "avatar": ":palm_tree:"
+    },
+    "client_config": {
+        "model": "xlarge",
```

Comment:
apparently with the `command-xlarge-20221108` is not doing great:
![Screenshot 2022-12-21 at 16 04 26](https://user-images.githubusercontent.com/13637091/208936580-8e657263-228c-4f2a-97cd-19f499fe31a2.png)

Reply:
wow that's really interesting 👀  -- we do `lstrip()` the generation so I guess the response itself is somehow blank 🤔. Alrighty, let's stick to xlarge for now. This is good to know though!

Reply:
The reason for this is that `command-xlarge-20221108` tends to start generation with `\n`, but since this persona includes that as a stop sequence, then nothing more is generated